docs/getting-started/designers/index.md

```diff
@@ -37,7 +38,7 @@ We currently have the following Figma libraries available:
         
         *You can have as many libraries as you want turned on but we suggest that you turn off the **Fabric core library** while migrating.*
         
-3. Turn on / or keep the **Fabric Icons library** turned on to get access to the Fabric icons
+3. Turn on the **Warp Icons library** to get access to the Warp icons
```

Comment:
We are still suggesting to use Fabric Icons on the dev side because we haven't yet moved icons to Warp. Is it safe to inform about Warp Icons library here then?